code/modules/mining/smelting.dm

```diff
@@ -17,119 +17,119 @@
 /datum/smelting_recipe/glass
 	name = "Glass"
 	ingredients=list(
-		MAT_GLASS = 1
+		MAT_GLASS = CC_PER_SHEET_GLASS
```

Comment:
Updating these to be CC_PER_SHEET does two things:

1. it completely bricks the recycling furnace, since you forgot to update that.
2. now all the numbers in the ore smelter are arbitrarily 3000x larger. This is just terrible from a usability perspective.